### PR TITLE
doc / lm improvement

### PR DESCRIPTION
**Pull request by JeremyKono**
Added the new Liquidity Mining strategy improvements.

image
image

### CODE REVIEW

File: content/strategies/liquidity-mining.mdx [hunk 1]
```diff
@@ -165,63 +167,111 @@ This expands the average spread depending on the value set. The predefined value
   response=">>> 1"
 />
 
-## Sample Configurations
+### `max_spread`
 
-Here's an example of how the liquidity mining strategy works.
+Set the maximum spread for your orders. Spreads will not go higher if you set a value for max spread. By default, this parameter is set to -1.
```
Comment (Nullably) [inline content/strategies/liquidity-mining.mdx:172]: Grammar error and description is not clear. 
Suggestion:

Sets the maximum spread for your orders. Spreads will not go higher than the value you set for this configuration.

File: content/strategies/liquidity-mining.mdx [hunk 2]
```diff
@@ -165,63 +167,111 @@ This expands the average spread depending on the value set. The predefined value
...
+
+<Prompt
+  prompt="What is the maximum spread? (Enter 1 to indicate 1% or -1 to ignore this setting)"
+  response=">>> 1"
+/>
+
+### `max_order_age`
+
+The `max_order_age` parameter allows you to set a specific duration when to reset your order's age. Click [here](/strategies/max-order-age/) to learn more about this parameter.
```
Comment (Nullably) [inline content/strategies/liquidity-mining.mdx:181]: The description is not clear. 
Suggestion:
Allows you to set a maximum lifespan of your orders (in seconds) at which the orders will certainly get canceled and recreated.

File: content/strategies/liquidity-mining.mdx [hunk 3]
```diff
@@ -165,63 +167,111 @@ This expands the average spread depending on the value set. The predefined value
...
+
+<Prompt
+  prompt="How often do you want to cancel and replace bids and asks (in seconds)"
```
Comment (Nullably) [inline content/strategies/liquidity-mining.mdx:184]: This is not the prompt for this config, please check.

**Review by Nullably — CHANGES_REQUESTED** [review]

**Comment by Nullably** [comment]
Why only `markets` table gets detailed explanation while other tables don't?

**Comment by JeremyKono** [comment]
Everything has been addressed in the latest commit

**Review by Nullably — APPROVED** [review]

**Pull request merged**